docs/Tutorials/ServerOperations/Overview.md

```diff
@@ -0,0 +1,81 @@
+# Overview
+
+Pode offers a suite of server management operations to provide granular control over server behavior. These operations include **suspending**, **resuming**, **restarting**, and **disabling/enabling** the server. Each operation is designed to address specific use cases, such as debugging, maintenance, or load management, ensuring the server can adapt to dynamic requirements without a complete shutdown.
+
+### Learn More About Server Operations
+
+- [**Suspending**](./Suspending/Overview.md): Temporarily pause server activities and later restore them without a full restart.
+- [**Restarting**](./Restarting/Overview.md): Multiple ways to restart the server, including file monitoring, scheduled restarts, and manual commands.
+- [**Disabling**](./Disabling/Overview.md): Block or allow new incoming requests without affecting the server’s state.
+
+---
+
+## Configuring Allowed Actions
+
+Pode introduces the ability to configure and control server behaviors using the `AllowedActions` section in the `server.psd1` configuration file. This feature enables fine-grained management of server operations, including suspend, restart, and disable functionality.
+
+### Default Configuration
+
+```powershell
+@{
+    Server = @{
+        AllowedActions = @{
+            Suspend = $true       # Enable or disable the suspend operation
+            Restart = $true       # Enable or disable the restart operation
+            Disable = $true       # Enable or disable the disable operation
+            DisableSettings = @{
+                RetryAfter = 3600                               # Default retry time (in seconds) for Disable-PodeServer
+                MiddlewareName = '__Pode_Midleware_Code_503'    # Name of the middleware scriptblock
+            }
+            Timeout = @{
+                Suspend = 30       # Maximum seconds to wait before suspending
+                Resume  = 30       # Maximum seconds to wait before resuming
+            }
+        }
+    }
+}
+```
+
+### Key Configuration Options
+
+1. **Suspend**: Enables or disables the ability to suspend the server via `Suspend-PodeServer`.
+2. **Restart**: Allows you to enable or disable server restarts.
+3. **Disable**: Controls whether the server can block new incoming requests using `Disable-PodeServer`.
+4. **DisableSettings**:
+   - `RetryAfter`: Specifies the default retry time (in seconds) included in the `Retry-After` header when the server is disabled.
+   - `MiddlewareName`: Defines the name of the middleware scriptblock responsible for handling `503 Service Unavailable` responses.
+5. **Timeout**:
+   - `Suspend`: Defines the maximum wait time (in seconds) for runspaces to suspend.
+   - `Resume`: Defines the maximum wait time (in seconds) for runspaces to resume.
```

Comment:
The sub-points, using `-`, need to be indented using 4 spaces, otherwise mkdocs will render them as further number-points:

```
4. **DisableSettings**:
    - `RetryAfter`: Specifies the default retry time (in seconds) included in the `Retry-After` header when the server is disabled.
    - `MiddlewareName`: Defines the name of the middleware scriptblock responsible for handling `503 Service Unavailable` responses.
5. **Timeout**:
    - `Suspend`: Defines the maximum wait time (in seconds) for runspaces to suspend.
    - `Resume`: Defines the maximum wait time (in seconds) for runspaces to resume.
```